driverapi/driverapi.go

```diff
@@ -74,6 +74,9 @@ type Driver interface {
 
 	// Type returns the the type of this driver, the network type this driver manages
 	Type() string
+
+	// IsBuiltIn returns true if it is a built-in driver
+	IsBuiltIn() bool
```

Comment:
Have you considered adding a `builtin` flag into the `initializer` structure instead of adding this new method ?

```
type initializer struct {
	fn      drvregistry.InitFunc
	ntype   string
+	builtin bool
}
```

Adding the new API is not really needed for this fix.